packages/design-system/src/components/slider/stories/index.js

```diff
@@ -37,27 +36,34 @@ const Container = styled.div`
 export default {
   title: 'DesignSystem/Components/Slider',
   component: Slider,
+  args: {
+    thumbSize: 24,
+    min: 0,
+    max: 100,
+    majorStep: 10,
+    minorStep: 1,
```

Comment:
Adding the asterisks here is really cool:

![image](https://user-images.githubusercontent.com/22185279/151243853-280d1a76-a455-49a1-a9dd-26dd72af8eea.png)

Wonder if storybook do that automatically with the `Slider` prop types

Reply:
The asterisk comes from the prop being required in propTyes! Took me a minute to figure out that one.